### DISCUSSION TITLE
Does Homer7 use DPDK/NFF-GO

### DISCUSSION THREAD

**Discussion by muhammadnaeemshahzad** [sipcapture/homer#461] [Q&A] [answered]
Does homer7 use DPDK/NFF-GO for fast packet processing? 
I would also like to know the performance benchmarks for Homer7 with or without DPDK support.

**Comment by lmangani** [comment, marked as answer]
Hi @muhammadnaeemshahzad 

Short anwer: No.

AF_PACKET is mostly used by our passive sniffing and packet indexing agents. DPDK features are only available in [Hepic](https://hepic.tel) deployments and in our Cubro EXA8 HEP Agent integration but not key to our design. There are no generic (as in misleading) benchmarks for Homer as unlike other platforms we do NOT store any packet/PCAPs to disk linearly but rather dissect and index protocols and transactions to database in realtime, performance is directly proportional to the amount of traffic content being processed vs. system specs and architecture.
